Makefile

```diff
@@ -94,13 +94,29 @@ deps:
 	cargo install --version 1.14.0 hyperfine
 	cargo install --version 0.9.49 cargo-nextest
 	cargo install --version 0.5.9 cargo-llvm-cov
-	pyenv install pypy3.7-7.3.9
-	pyenv global pypy3.7-7.3.9
+	cargo install --version 0.11.0 wasm-pack
+	pyenv install pypy3.9-7.3.9
+	pyenv local pypy3.9-7.3.9
 	pip install -r requirements.txt
-	pyenv install 3.7.12
-	pyenv global 3.7.12
+	pyenv install 3.9.15
```

Comment:
We can use the `-s` option for `pyenv install`, so it skips installation if the version is already installed. Right now it prompts the user for confirmation.

Reply:
Thanks Mega!
Didn't know that